tests/output_filters/related_files_test.py

```diff
@@ -1,59 +1,89 @@
+        if expected_usernames:
+            assert_equal_sorted(expected_usernames, self._output_filter.usernames)
+        if expected_is_related:
+            self.assert_key_added_to_blob('osxcollector_related', expected_is_related, input_blobs, output_blobs)
+        return output_blobs
+
+
+class CreateTermsTest(RelatedFilesFilterTest):
+
+    """Focuses on testing that terms are properly created."""
 
     def test_single_term(self):
         initial_terms = ['one_word']
         expected = ['one_word']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_multi_terms(self):
         initial_terms = ['one_word', 'pants', 'face']
         expected = ['one_word', 'pants', 'face']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_split_terms(self):
         initial_terms = ['/ivanlei/source/osxcollector']
         expected = ['ivanlei', 'source', 'osxcollector']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_whitelist_terms(self):
         initial_terms = ['/Users/ivanlei/source/osxcollector', '/Users/ivanlei/virtual_envs/osxcollector/bin/python']
         expected = ['ivanlei', 'source', 'osxcollector', 'virtual_envs']
-        self._test_create_terms(initial_terms, expected)
+        self._run_test(initial_terms=initial_terms, expected_terms=expected)
 
     def test_whitelist_username_terms(self):
         initial_terms = ['/Users/ivanlei/source/osxcollector', '/Users/ivanlei/virtual_envs/osxcollector/bin/python']
         expected = ['source', 'osxcollector', 'virtual_envs']
         blob = {'osxcollector_username': 'ivanlei'}
 
-        self._test_create_terms(initial_terms, expected, blob)
+        self._run_test(input_blobs=[blob], initial_terms=initial_terms, expected_terms=expected, expected_usernames=['ivanlei'])
 
-    def _test_create_terms(self, initial_terms, expected_terms, blob=None):
-        output_filter = RelatedFilesFilter(when=when_anytime, initial_terms=initial_terms)
-        if blob:
-            output_filter.filter_line(blob)
-        output_filter.end_of_lines()
-        T.assert_equal(sorted(expected_terms), sorted(output_filter.terms))
 
+class FindUserNamesTest(RelatedFilesFilterTest):
 
-class FindUserNamesTest(T.TestCase):
+    """Focuses on ensuring that usernames are found so they can be ignored as terms."""
 
     def test_find_username(self):
         blob = {'osxcollector_username': 'bob'}
-        expected = ['bob']
-
-        output_filter = RelatedFilesFilter(when=when_anytime)
-        output_filter.filter_line(blob)
-        T.assert_equal(sorted(expected), sorted(output_filter._usernames))
+        self._run_test(input_blobs=[blob], expected_usernames=['bob'])
```

Comment:
Super minor, but you might want to move "['bob']" under "blob = {..." for consistency.